packages/riipen-ui/src/components/Tabs.jsx

```diff
@@ -86,8 +86,10 @@ class Tabs extends React.Component {
     const childrenWithProps = React.Children.map(children, child => {
       if (!child) return null;
 
+      const active = child.props.active || child.props.value === value;
```

Comment:
Feel like we should have some way of preventing multiple tabs in a list from being `active`, but not sure how to do that. Think we could do that in a separate ticket

Reply:
So this worked the way you described before as active was always being overwritten. Maybe i should name the prop `forceActive` or something